src/app/forms/components/weights/weights.component.html

```diff
@@ -0,0 +1,95 @@
+<ng-container *ngIf="isEditable">
+  <form [formGroup]="form">
+    <table class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">
+        Axle weights (kg)
+      </caption>
+
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header"></th>
+          <th scope="col" class="govuk-table__header">Kerb</th>
+          <th scope="col" class="govuk-table__header">Laden</th>
+          <th scope="col" class="govuk-table__header">GB max</th>
+          <th scope="col" class="govuk-table__header">Design</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <ng-container formArrayName="axles">
+          <tr class="govuk-table__row" *ngFor="let axle of axlesFormArray.controls; let i = index">
+            <ng-container [formGroupName]="i">
+              <th scope="row" class="govuk-table__header">Axle {{ axle.value.axleNumber }}</th>
+              <ng-container formGroupName="weights">
+                <td class="govuk-table__cell">
+                  <app-text-input name="foo" formControlName="kerbWeight"></app-text-input>
+                </td>
+                <td class="govuk-table__cell"><app-text-input name="foo" formControlName="ladenWeight"></app-text-input></td>
```

Comment:
`name = "foo"` ? Seeing as this name is used to create the `id` property of the underlying element, I believe, they should be unique ideally

Reply:
Lmao I forgot to change this, so so sorry, dunno how I didn't spot this